spice_ev/strategies/balanced_market.py

```diff
@@ -8,11 +8,10 @@
 class BalancedMarket(Strategy):
     """ Price oriented charging at times of low energy price. """
     def __init__(self, components, start_time, **kwargs):
-        self.PRICE_THRESHOLD = 0.001  # EUR/kWh
         self.HORIZON = 24  # maximum number of hours ahead
 
         super().__init__(components, start_time, **kwargs)
-        assert len(self.world_state.grid_connectors) == 1, "Only one grid connector supported"
+        # assert len(self.world_state.grid_connectors) == 1, "Only one grid connector supported"
```

Comment:
why is this a comment and not removed entirely?

Reply:
was just a temporary solution. See PR #187